prisma/schema.prisma

```diff
@@ -62,6 +62,7 @@ model TelegramGroup {
   telegramId BigInt         @map("telegram_id")
   threadId   BigInt?        @map("thread_id")
   source     TelegramSource
+  postInfo   Boolean        @default(false) @map("post_info")
```

Comment:
Add postInfo to this endpoint 
![image](https://github.com/fictadvisor/fictadvisor-api/assets/116638814/5241b900-5b39-4398-bf81-db8cb1e00f40)
![image](https://github.com/fictadvisor/fictadvisor-api/assets/116638814/ab232310-239a-4d74-bb1d-8459732ada7e)